### ISSUE TITLE
Consider adding Save and Exit function

### ISSUE THREAD

**Issue by SusanBrown** [cwrc/NERVE#12]
I've drafted this for the splash page:

Note that if you reopen NERVE using the same computer and browser, the file you were using will reload in its most recent state, if you have not cleared your browsing data including cached files. However, you are strongly advised to use Save function after each work session.

If this is true, then I wonder if we should have a separate Save and Exit function that does a reset (except for the Schema selection). It's great that NERVE retains what you've done, but I fear that people will rely on it too much and/or get confused or misled about the current version of the file.

**Comment by ilovan** [comment]
This has cropped up as a suggestion from the RA who has tested NERVE recently, so I would say it's something people should be made aware of and  be able to exercise their choice about.

**Comment by susanirenebrown** [comment]
Don’t understand the wording here: are you suggesting we poll people?

**Comment by ilovan** [comment]
No, I meant that this is something we should definitely consider implementing: People should be aware of the fact that  they need to clear they cache if they want to keep their document private. Also, they should have the "save and exit" option available.

**Comment by SusanBrown** [comment]
OK. Agree. Please suggest specific wording/sequence, i.e. would the “Save and Exit” option spawn a popup that would ask if they want the cache cleared or want to return to their document later on the same machine?


On Nov 24, 2017, at 4:37 PM, Mihaela Ilovan <notifications@github.com<mailto:notifications@github.com>> wrote:


No, I meant that this is something we should definitely consider implementing: People should be aware of the fact that they need to clear they cache if they want to keep their document private. Also, they should have the "save and exit" option available.
